docs/updating/5_update_2.2.md

```diff
@@ -1,6 +1,8 @@
 # Update database to v2.2
 
-If you are updating from a version prior to v2.2, you have to implement all the changes [from v1.13](5_update_1.13.md) before following the steps below.
+!!! caution
+
+    If you are updating from a version prior to v2.2, you have to implement all the changes [from v1.13](5_update_1.13.md) before following the steps below.
```

Comment:
```suggestion
    When you are updating from a version prior to v2.2, you must implement all the changes [from v1.13](5_update_1.13.md) before you proceed to the steps below.
```
Let's make it a strong statement to halt them in their tracks.

If you agree, make similar updates to the changes below.